packages/lib-classifier/src/components/Classifier/components/TaskArea/components/Tasks/components/SingleChoiceTask/SingleChoiceTask.js

```diff
@@ -62,7 +62,7 @@ class SingleChoiceTask extends React.Component {
     }
 
     return (
-      <StyledFieldSet
+      <StyledBox
```

Comment:
Same here, and I don't think `Box` has `border`, `margin` or `padding` set by default so it doesn't need to be a styled component

Reply:
Removed the `border`, `margin`, and `padding`, but kept stylings for `img` and `svg` for single/multiple question tasks with images included in option labels, I think an example would be Galaxy Zoo.

Reply:
Though additional styling work is necessary for single/multiple question tasks with images included in option labels to look good, this keeps the basic selectors/styling that existed before any drawing task work, that was previously included in the general task input.